sdk/synapse/azure-synapse-artifacts/azure/synapse/__init__.py

```diff
@@ -1 +1 @@
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```

Comment:
It is better to add some test case

Reply:
You can refer to guidance https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test and existing testcase https://github.com/Azure/azure-sdk-for-python/tree/e0c7db941d9f512bab01a75c4a43d26cbfe946f2/sdk/synapse/azure-synapse-artifacts/tests.

Reply:
Sure. Will add test case on next release.